lib/Doctrine/Migrations/Configuration/Configuration.php

```diff
@@ -174,9 +175,23 @@ public function isDatabasePlatformChecked(): bool
         return $this->checkDbPlatform;
     }
 
-    public function setMigrationOrganization(string $migrationOrganization): void
+    /**
+     * @param string | bool $migrationOrganization
```

Comment:
instead of allowing both `string` and `bool`, what about having a sting costant  `none` that would reset the migration organization?

Reply:
I would be OK with that.